docker/stubs/flac/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ouspg/libfuzzer-base
+
+MAINTAINER https://github.com/ouspg/libfuzzerification
+ENV TARGET 'flac'
+
+#Install dependencies and fetch the source.
+
+RUN apt-get update && apt-get upgrade -y && apt-get install -y libflac-dev pkg-config libtool-bin && apt-get build-dep libflac-dev -y
+RUN cd /src && git clone https://git.xiph.org/flac.git
+ADD flac_fuzzer.c /src/flac/
+
+ADD ${TARGET}_fuzzer.c /src/$TARGET/
```

Comment:
This line does the same thing as `ADD flac_fuzzer.c /src/flac/` above?

Also using $TARGET (or ${TARGET}, notice inconsistent syntax) instead of just "flac" just leads to probably unneccessary obfuscation? It's must easier to grep "flac_fuzzer.c" when trying to figure out where it's used.